src/components/Navbar/Components/ProjectCard.tsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from "react";
+import { Axios } from "../../../utils/axios";
+import { API } from "../../../utils/usedApi";
+import ProjectItemCard from "./ProjectItemCard";
+
+type ProjectCardProps = {
+  color: string;
+  name: string;
+};
+
+type Project = {
+  id: number;
+  name: string;
+  color: string;
+};
+
+function ProjectCard({ name, color }: ProjectCardProps) {
+  const [projects, setProjects] = useState<Project[]>([]);
+
+  useEffect(() => {
+    const userId = sessionStorage.getItem("userId");
+    if (!userId) return;
+
+    Axios.get(`${API.getProject}/${userId}`)
+      .then((response) => {
+        console.log(response.data);
+        setProjects(response.data);
+      })
+      .catch((err) => {
+        console.log(err);
+        setProjects([]);
```

Comment:
Why are you setting projects []? What about if we already had 5 projects and got an error while adding 6th? Will you remove the existing ones?

Reply:
Sifu did you mean this:  ` setProjects((prevProjects) => [...prevProjects, newProject])`